model/BUILD.bazel

```diff
@@ -186,14 +186,14 @@ format_import_proto_library(
 java_lite_proto_library(
     name = "exploration_java_proto_lite",
     deps = [":exploration_proto"],
+    visibility = ["//visibility:public"],
```

Comment:
In general we should avoid doing this. Any reason why we want to make this public?

Reply:
@vinitamurthi, are you saying that we should be explicit about the visibility?

Reply:
Does https://github.com/oppia/oppia-android/pull/2699#discussion_r575296434 help clarify the need @vinitamurthi or do you have a separate concern?